src/test/data/draft/claimDraft.ts

```diff
@@ -21,9 +21,11 @@ export const claimDraft = {
   },
   claimant: {
     payment: {
-      id: 12,
+      reference: '123',
+      dateCreated: 12345,
       amount: 10000,
-      state: { status: 'success' }
+      status: 'Success',
+      links: { nextUrl: { href: 'any href', method: 'POST' } }
```

Comment:
That is not right, initiate payment response has `_links` and `next_url` properties.